cmd/subnetcmd/deploy_test.go

```diff
@@ -161,7 +161,7 @@ func TestCheckForInvalidDeployAndSetAvagoVersion(t *testing.T) {
 			networkErr:      nil,
 			desiredRPC:      19,
 			desiredVersion:  testLatestAvagoVersion,
-			expectError:     true,
+			expectError:     false,
```

Comment:
is it a separate fix for e2e?  and it's not relevant to `teleporter` if I understand this code correctly

Reply:
this is a fix for unit testing. while testing teleporter and upgrading to durango issues were found on compatibility checks, so a change was made to CheckForInvalidDeployAndGetAvagoVersion:
- now removes the rc part of the provided avago version for comparisons
- if no avago version was found for a given subnet, it returns pre release (which should be most probablty situation of avago prerelease using a new rpc version not found in compatibility file)